addons/account/wizard/account_automatic_entry_wizard.py

```diff
@@ -496,5 +496,6 @@ def _format_strings(self, string, move, amount=None):
             link=self._format_move_link(move),
             date=format_date(self.env, move.date),
             new_date=self.date and format_date(self.env, self.date) or _('[Not set]'),
+            account_source_name=account_source_name or '',
```

Comment:
I would say that you can directly put the '' in the function declaration instead of the None.
Except that, LGTM